bgpd/bgp_mplsvpn.c

```diff
@@ -1555,7 +1559,7 @@ void vpn_handle_router_id_update(struct bgp *bgp, bool withdraw,
 				 bool is_config)
 {
 	afi_t afi;
-	int debug;
+	int debug = BGP_DEBUG(vpn, VPN_LEAK_LABEL);
```

Comment:
Just curious: Why Leak label here?

Reply:
Yes, it is not correct.
We will change it to:
```c
	debug = (BGP_DEBUG(vpn, VPN_LEAK_TO_VRF) |
		     BGP_DEBUG(vpn, VPN_LEAK_FROM_VRF));
```

Reply:
Done